src/main/java/com/ning/compress/lzf/impl/UnsafeChunkEncoderBE.java

```diff
@@ -83,7 +88,7 @@ protected int tryCompress(byte[] in, int inPos, int inEnd, byte[] out, int outPo
             ++inPos;
         }
         // try offlining the tail
-        return _handleTail(in, inPos, inEnd+4, out, outPos, literals);
+        return _handleTail(in, inPos, inEnd+TAIL_LENGTH, out, outPos, literals);
```

Comment:
I hope this is correct; to me it looked like this undoes the `-= TAIL_LENGTH` at the beginning of the method:
https://github.com/ning/compress/blob/b206cc90d673c4230dc65ea00342f77e742640a8/src/main/java/com/ning/compress/lzf/impl/UnsafeChunkEncoderBE.java#L34